src/client/src/rt-theme/themes.ts

```diff
@@ -71,25 +73,10 @@ export interface ColorPair {
   textColor?: string
 }
 
-export const themes = {
-  light: createTheme(colors.light, colors.accents),
-  dark: createTheme(colors.dark, colors.accents, (theme: Theme) => ({
-    button: {
-      ...theme.button,
-      secondary: {
-        ...theme.button.secondary,
-        textColor: theme.primary.base
-      }
-    }
-  }))
-}
+type ThemeModifier = (original: GeneratedTheme) => GeneratedTheme
 
-export function createTheme(
-  { primary, secondary }: CorePaletteMap,
-  accents: AccentPaletteMap,
-  modify: (originalTheme) => object = (theme: Theme) => theme
-) {
-  const newTheme = {
+function generateTheme({ primary, secondary }: CorePaletteMap, accents: AccentPaletteMap) {
```

Comment:
Use arrow syntax with implicit object return

Reply:
done. But not sure why its important?